tabular/src/autogluon/tabular/task/tabular_prediction/predictor.py

```diff
@@ -598,34 +601,63 @@ def feature_importance(self, dataset=None, model=None, features=None, feature_st
             If `subsample_size=None` or `dataset` contains fewer than `subsample_size` rows, all rows will be used during computation.
             Larger values increase the accuracy of the feature importance scores.
             Runtime linearly scales with `subsample_size`.
+        time_limit : float, default = None
+            Time in seconds to limit the calculation of feature importance.
+            If None, feature importance will calculate without early stopping.
+            A minimum of 1 full shuffle set will always be evaluated. If a shuffle set evaluation takes longer than `time_limit`, the method will take the length of a shuffle set evaluation to return regardless of the `time_limit`.
+        num_shuffle_sets : int, default = None
+            The number of different permutation shuffles of the data that are evaluated.
+            Larger values will increase the quality of the importance evaluation.
+            It is generally recommended to increase `subsample_size` before increasing `num_shuffle_sets`.
+            Defaults to 1 if `time_limit` is None or 10 if `time_limit` is specified.
+            Runtime linearly scales with `num_shuffle_sets`.
+        include_confidence_band: bool, default = True
+            If True, will include output columns 'p99_high' and 'p99_low' which indicates that the true feature importance will be between 'p99_high' and 'p99_low' 99% of the time.
+            Increasing `subsample_size` and `num_shuffle_sets` will tighten the band.
         silent : bool, default = False
-            Whether to suppress logging output
+            Whether to suppress logging output.
 
         Returns
         -------
-        Pandas `pandas.Series` of feature importance scores.
-
+        Pandas `pandas.DataFrame` of feature importance scores with 6 columns:
+            index: The feature name.
+            'importance': The estimated feature importance score.
+            'stddev': The standard deviation of the feature importance score. If NaN, then not enough num_shuffle_sets were used to calculate a variance.
+            'p_value': The probability that the feature's true importance score is less than or equal to 0. Feature with p_values > 0.9 are likely harmful.
+            'n': The number of shuffles (samples) performed.
```

Comment:
may be clearer as:
"The number of shuffles performed to estimate importance score (corresponds to sample-size used to determine confidence interval for true score)"

Reply:
Added